src/vm/comutilnative.cpp

```diff
@@ -125,15 +125,6 @@ FCIMPL1(FC_BOOL_RET, ExceptionNative::IsImmutableAgileException, Object* pExcept
 }
 FCIMPLEND
 
-FCIMPL1(FC_BOOL_RET, ExceptionNative::IsTransient, INT32 hresult)
-{
-    FCALL_CONTRACT;
-
-    FC_RETURN_BOOL(Exception::IsTransient(hresult));
```

Comment:
Exception::IsTransient is still used elsewhere in the runtime?

Reply:
Yes, it is still used in the runtime.